examples/http-trace-logs/http_trace_logs.bal

```diff
@@ -5,17 +5,17 @@ endpoint http:Client clientEP {
     url: "http://httpstat.us"
 };
 
-@http:ServiceConfig {basePath:"/hello"}
-service<http:Service> helloWorld bind {port:9090} {
+@http:ServiceConfig {basePath: "/hello"}
+service<http:Service> helloWorld bind {port: 9090} {
     @http:ResourceConfig {
-        methods:["GET"],
-        path:"/"
+        methods: ["GET"],
+        path: "/"
     }
-    sayHello (endpoint caller, http:Request req) {
+    sayHello(endpoint caller, http:Request req) {
         var resp = clientEP->forward("/200", req);
         match resp {
             http:Response response => caller->respond(response)
-                                            but { error e => log:printError("Failed to respond to caller", err = e) };
+            but { error e => log:printError("Failed to respond to caller", err = e) };
```

Comment:
I think it would be clear if we have this like below:
```ballerina
http:Response response => caller->respond(response) but { 
                                error e => log:printError("Failed to respond to caller", err = e) 
                          };
````